app/serve.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const compression = require('compression')
+const path = require('path');
+const app = express();
+
+app.use(express.static(path.join(__dirname, '..', 'dist')));
+app.use(compression())
+
+app.get('/runtime-config.js', function (req, res) {
```

Comment:
it might be nice to also add an `/status` endpoint to this server

Reply:
That's a good idea